src/prefetch.mjs

```diff
@@ -104,20 +95,14 @@ const supportedPrefetchStrategy = support(`prefetch`)
  * @param {Boolean} isPriority - if is "high" priority
  * @return {Object} a Promise
  */
-function prefetcher(url, isPriority) {
+function prefetcher(url, isPriority, conn) {
   if (preFetched[url]) {
     return;
   }
 
-  if ('connection' in navigator) {
-    // Don't prefetch if the user is on 2G...
-    if ((navigator.connection.effectiveType || '').includes('2g')) {
-      return;
-    }
-    // Don't prefetch if Save-Data is enabled...
-    if (navigator.connection.saveData) {
-      return;
-    }
+  if (conn = navigator.connection) {
+    // Don't prefetch if the user is on 2G. or if Save-Data is enabled..
+    if ((conn.effectiveType || '').includes('2g') || conn.saveData) return;
```

Comment:
I know you asked for this to be reverted in the last attempt, but perhaps the `conn` abbreviation preserves the clarity. It did for me.

Reply:
I'm okay with us switching up to this abbreviation. I agree it reads a little better than the last version 👍